ethereum/graphqlrpc/src/test/java/tech/pegasys/pantheon/ethereum/graphqlrpc/EthGraphQLRpcHttpBySpecTest.java

```diff
@@ -108,14 +109,14 @@ private void graphQLRPCCall(final String name) throws IOException {
 
     importBlocks(1, BLOCKS.size());
     try (final Response resp = client.newCall(request).execute()) {
-      final int expectedStatusCode = spec.getInteger("statusCode");
-      assertThat(resp.code()).isEqualTo(expectedStatusCode);
-
       final JsonObject expectedRespBody = spec.getJsonObject("response");
       final String resultStr = resp.body().string();
 
       final JsonObject result = new JsonObject(resultStr);
       assertThat(result).isEqualTo(expectedRespBody);
+
+      final int expectedStatusCode = spec.getInteger("statusCode");
+      assertThat(resp.code()).isEqualTo(expectedStatusCode);
```

Comment:
Why moving this at the end?

Reply:
So when an error occurs the response body shows up in the junit report.